corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -6,15 +6,17 @@ hqDefine("cloudcare/js/formplayer/menus/collections", function () {
 
     var MenuSelect = Backbone.Collection.extend({
         commonProperties: [
-            'title',
-            'type',
+            'appId',
+            'appVersion',
+            'breadcrumbs',
             'clearSession',
             'notification',
-            'breadcrumbs',
-            'appVersion',
-            'appId',
             'persistentCaseTile',
+            'requireSearch',
+            'searchText',
             'tiles',
+            'title',
+            'type',
             'selections',
```

Comment:
it took me way too long to realize you put these in alphabetical order

Reply:
ah, sorry, I thought about pulling that out as a separate commit and got lazy